src/api/app/components/request_decision_component.html.haml

```diff
@@ -16,41 +16,52 @@
         - if policy(Comment.new(commentable:@bs_request)).create?
           = submit_tag 'Add comment', class: 'btn btn-primary me-2', data: { disable_with: 'Creating comment...' }, name: 'commented'
         - if policy(@bs_request).revoke_request?
-          = submit_tag('Revoke request', name: 'revoked', class: 'btn btn-danger me-2',
+          = submit_tag('Revoke request', name: 'revoked', class: 'btn btn-outline-danger me-2',
                         title: 'Cancel the request', data: { confirm: "Revoke this request?" })
         - if policy(@bs_request).decline_request?
-          = submit_tag('Decline request', name: 'declined', class: 'btn btn-danger me-2',
+          = submit_tag('Decline request', name: 'declined', class: 'btn btn-outline-danger me-2',
                         title: 'Reject the changes.', data: { confirm: "Decline this request?\n\n#{@package_maintainers_hint}" })
         - if policy(@bs_request).accept_request?
           %span.dropdown.me-2
             - unless show_forward? || show_add_creator_as_maintainer?
-              = submit_tag('Accept request', name: 'accepted', class: 'btn btn-primary me-2',
+              = submit_tag('Accept request', name: 'accepted', class: 'btn btn-outline-secondary me-2',
```

Comment:
Why not at least `btn-outline-primary`? It's arguable that accepting is even  *more* important than commenting I would say...